aws_prom_exporter/modules/config/__init__.py

```diff
@@ -29,8 +32,13 @@
         'file_sd_path': '/etc/prometheus/file_sd',
         'exporter': {
             'mysqld': {
-                'docker_image': 'prom/mysqld-exporter'
-            }
+                'docker_image': 'prom/mysqld-exporter',
+                'docker_arguments': ''
+             },
+            'redis': {
+                'docker_image': 'oliver006/redis_exporter',
+                'docker_arguments': '--redis.addr=  '
```

Comment:
This should be empty

```suggestion
                'docker_arguments': ''
```

Reply:
Ithough it was required to avoid it to check localhost.